apps/dashboard/app/models/user_configuration.rb

```diff
@@ -99,6 +105,16 @@ def public_url
     path.start_with?('/') ? Pathname.new(path) : Pathname.new('/public')
   end
 
+  # Filtering is controlled with NavConfig.categories_whitelist? unless the configuration property categories is defined.
+  # If categories are defined, filter_nav_categories? will always be true.
+  def filter_nav_categories?
+    fetch(:categories, nil).nil? ? NavConfig.categories_whitelist? : true
+  end
+
+  def categories
+    fetch(:categories, NavConfig.categories)
+  end
```

Comment:
@adaybujeda I pulled this in (and perhaps gave you the example) without thinking.  This should be `nav_categories` not just `categories`.  This is the user facing configuration - so it should indicate that it's specific to the navigation bar, and not, well you can't really tell what `categories` would apply to.

Can you submit the change?

Reply:
Sure - Will do

Reply:
created new PR to update the name: https://github.com/OSC/ondemand/pull/2257